c2rust-transpile/src/translator/simd.rs

```diff
@@ -428,16 +428,9 @@ impl<'c> Translation<'c> {
         let param_translation =
             self.convert_exprs(ctx.used(), &[first_expr_id, second_expr_id, mask_expr_id])?;
         param_translation.and_then(|params| {
-            let mut params = params.into_iter();
-            let first = params
-                .next()
-                .ok_or("Missing first param in convert_shuffle_vector")?;
-            let second = params
-                .next()
-                .ok_or("Missing second param in convert_shuffle_vector")?;
-            let third = params
-                .next()
-                .ok_or("Missing third param in convert_shuffle_vector")?;
+            let [first, second, third]: [_; 3] = params
+                .try_into()
+                .expect("`convert_shuffle_vector` is expected to have exactly 3 parameters");
```

Comment:
```suggestion
                .expect("`convert_shuffle_vector` must have exactly 3 parameters");
```

Use the same error message as the others.  `parameters` instead of `arguments` is okay as the variable is called `params`, but the rest should be the same.